contracts/StablecoinConverter.sol

```diff
@@ -216,7 +216,7 @@ contract StablecoinConverter is EpochTokenLocker {
     }
 
     function grantRewardToSolutionSubmitter() internal {
```

Comment:
I don't see this being used anywhere by the contract and it appears to be internal. Was this intended? Is the usage of this coming later? It doesn't seem to contain any logic apart from dividing the fees by two and using the feeToken. Is this really necessary?

Also there does not appear to be any unit test for this function... Do we still have full coverage?

Reply:
> Do we still have full coverage?

We can't merge without full coverage

Reply:
The coverage tool spotted this as well in the first commit. Somehow you were looking only at the first commit and not at the second...

Reply:
Oh...